File: framework/languages/java/samples/java/GameQuest/Client/src/main/java/systems/zlink/samples/gamequest/client/configuration/SampleTimings.java
package systems.zlink.samples.gamequest.client.configuration;

import java.time.Duration;

public final class SampleTimings {
    public static final Duration ConnectTimeout = Duration.ofSeconds(10);
    public static final Duration RequestTimeout = Duration.ofSeconds(10);
    public static final Duration NotifyTimeout = Duration.ofSeconds(20);
    public static final Duration PollDelay = Duration.ofMillis(100);

    private SampleTimings() {
    }
}

File: framework/languages/java/samples/kotlin/GameQuest/Client/src/main/kotlin/systems/zlink/samples/kotlin/gamequest/client/configuration/SampleTimings.kt
package systems.zlink.samples.kotlin.gamequest.client.configuration

import java.time.Duration

object SampleTimings {
    val ConnectTimeout: Duration = Duration.ofSeconds(10)
    val RequestTimeout: Duration = Duration.ofSeconds(10)
    val AssertAttemptTimeout: Duration = Duration.ofSeconds(2)
    val PollDelay: Duration = Duration.ofMillis(100)
    val NotifyTimeout: Duration = Duration.ofSeconds(20)
}
